### PR TITLE
Discussion initial draft

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

Here is an effort to write a discussion! I have never written a discussion for a lit review. VERY open to additions and feedback from anyone who is interested!

Some things we may want to address:
- I gave a nod the open source approach/technology at the end -- do we want to dig more into that?
- Does this hit all of the major topics that we should cover in the discussion?
- @cbrueffer I think you are more passionate about diagnostics than anyone else -- do you have thoughts on what should be added?

### Suggested reviewers (optional)

@cgreene @SiminaB @RLordan 
@juliettemarie0405 @shiktadas @ipsitak @catherinedawn in case you have feedback on the social factors sections!

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/50.discussion.md [hunk 1]
```diff
@@ -1,79 +1,96 @@
...
+
+Given that a large number of clinical trials are currently in progress, more information about the potential of these and other therapeutics should become available over time.
+This information, combined with advances in understanding the molecular structure and viral pathogenesis of SARS-CoV-2, may lead to a more complete understanding of how the virus affects the human host and what strategies can improve outcomes.
+To date, investigations of potential therapeutics for COVID-19 have focused primarily on repurposing existing drugs.
+This approach is necessary given the urgency of the situation as well as the extensive time required for developing and testing new therapies.
+However, in the long-term, new drugs specific for treatment of COVID-19 may also enter development.
+Development of novel drugs is likely to be guided by what is known about the pathogenesis and molecular structure of SARS-CoV-2.
+For example, understanding the structures of SARS-CoV-2 components may allow for the development of small molecule inhibitors of those components.
```
Comment (RLordan) [inline content/50.discussion.md:85]:
```suggestion
For example, understanding the various structural components of SARS-CoV-2 may allow for the development of small molecule inhibitors of those components.
```

File: content/50.discussion.md [hunk 2]
```diff
@@ -1,79 +1,96 @@
...
 While still nascent, work in this area is promising.
+Over the longer term, this approach and others may lead to the development of novel therapeutics specifically for COVID-19 and SARS-CoV-2.
 
 ### Social Factors Influencing COVID-19 Outcomes
 
+In addition to understanding the fundamental biology of the SARS-CoV-2 virus and COVID-19, it is critial to consider how the broader environment can influence both COVID-19 outcomes and efforts to develop and implement treatments for the disease.
 The evidence clearly indicates that social environmental factors are critical determinants of individuals' and communities' risks related to COVID-19.
 There are distinct components to COVID-19 susceptibility, and an individual's risk can be elevated at one or all stages from exposure to recovery/mortality: an individual may be more likely to be exposed to the virus, more likely to get infected once exposed, more likely to have serious complications once infected, be less likely to receive adequate care once they are seriously ill.
```
Comment (RLordan) [inline content/50.discussion.md:95]:
```suggestion
There are distinct components to COVID-19 susceptibility, and an individual's risk can be elevated at one or all stages from exposure to recovery/mortality: an individual may be more likely to be exposed to the virus, more likely to get infected once exposed, more likely to have serious complications once infected, and be less likely to receive adequate care once they are seriously ill.
```

**Review by RLordan — COMMENTED** [review]
The discussion looks very good to me. Only some minor things pointed out. I like the addition of the social aspects at the end.

**Comment by RLordan** [comment]
The only other thing I think of is to include a line (but maybe I missed it) on the nutraceuticals. Just an acknowledgment of the ongoing work, as it does take up a few pages of the manuscript.

File: content/50.discussion.md [hunk 3]
```diff
@@ -1,79 +1,96 @@
...
+Early interest in the drugs hydroxychloroquine and chloroquine yielded no promising results from studies with robust experimental designs.
+The experimental drug remdesivir, which was developed for ebola, has received enough support from early analyses to receive FDA approval, but results have been mixed.
+The potential for other drugs, such as tocilizumab, to reduce recovery time remains unclear, but some early results were promising.
+
+### Additional Therapeutics of Interest
+
+Given what is currently known about these therapeutics for COVID-19, a number of related therapies beyond those explored above may also prove to be of interest.
+For example, the demonstrated benefit of dexamethasone and the ongoing potential of tocilizumab for treatment of COVID-19 suggests that other anti-inflammatory agents might also hold value for the treatment of COVID-19.
+Given that current evidence about treating COVID-19 with dexamethasone suggests that the need to curtail the “cytokine storm” inflammatory response to the virus can transcend the risks of immunosuppression, exploration of more anti-inflammatory agents may be warranted.
```
[the remark this answers is not in the document]
Comment (rando2) [reply content/50.discussion.md:45]:
```suggestion
Given that current evidence about treating COVID-19 with dexamethasone suggests that the need to curtail the cytokine storm inflammatory response to the virus can transcend the risks of immunosuppression, exploration of more anti-inflammatory agents may be warranted.
```

File: content/50.discussion.md [hunk 4]
```diff
@@ -1,79 +1,96 @@
...
+While dexamethasone is considered widely available and generally affordable, the high costs of biologics such as tocilizumab therapy may present obstacles to wide-scale distribution of this drug if it proves of value.
+At the doses used for rheumatoid arthritis patients, the cost for tocilizumab ranges from $179.20 to $896 per dose for the IV form and $355 for the pre-filled syringe [@url:https://www.ncbi.nlm.nih.gov/books/NBK349513/table/T43/].
+There are several anti-inflammatory agents used for the treatment of autoimmune diseases that may also be able to counter the effects of the “cytokine storm” induced by the virus, some of which, such as cyclosporine, are likely to be more cost-effective and readily available than biologics [@url:https://escholarship.umassmed.edu/meyers_pp/385].
```
Comment (rando2) [inline content/50.discussion.md:48]:
```suggestion
There are several anti-inflammatory agents used for the treatment of autoimmune diseases that may also be able to counter the effects of the cytokine storm induced by the virus, some of which, such as cyclosporine, are likely to be more cost-effective and readily available than biologics [@url:https://escholarship.umassmed.edu/meyers_pp/385].
```

**Review by rando2 — COMMENTED** [review]
Thank you so much @RLordan!

**Comment by rando2** [comment]
> The only other thing I think of is to include a line (but maybe I missed it) on the nutraceuticals. Just an acknowledgment of the ongoing work, as it does take up a few pages of the manuscript.

@RLordan I think that is a great idea! Is this something you would want to add a few sentences about? Otherwise I can try to put something together! I initially thought the same thing but then forgot to do it!

File: content/20.treatments.md [hunk 5]
```diff
@@ -390,6 +390,9 @@ Additionally, the production of both catecholamines and glucocorticoids is assoc
...
 In this model, dexamethasone reduces inflammation by stimulating the mechanism used to reduce inflammation following a threat such as immune challenge.
+Immunosuppressive drugs such as steroids are typically contraindicated in the setting of infection [@doi:10.1177/2040622313485275], but because COVID-19 results in hyperinflammation that appears to contribute to mortality via lung damage, immunosuppression may be a helpful approach to treatment [@doi:10/ggnzmc].
+The decision of whether and/or when to counter hyperinflammation with immunosuppression in the setting of COVID-19 was an area of intense debate, as the risks of inhibiting antiviral immunity needed to be weighed against the beneficial anti-inflammatory effects [@doi:10/ggq8hs].
+As a result, guideliens early in the pandemic typically recommended avoiding treating COVID-19 patients with corticosteroids such as dexamethasone [@doi:10/ggks86].
```
Comment (rando2) [inline content/20.treatments.md:395]:
```suggestion
As a result, guidelines early in the pandemic typically recommended avoiding treating COVID-19 patients with corticosteroids such as dexamethasone [@doi:10/ggks86].
```

**Comment by RLordan** [comment]
Do you need me to approve @rando2 or waiting for more reviewers?

**Comment by rando2** [comment]
@RLordan if you don't mind, that would be awesome! I think today we're at a point where we can ask people to start reviewing the manuscript in preparation to submit :)

**Review by RLordan — APPROVED** [review]
Great to hear @rando2, stellar work by all the Greene team & Co! Thank you.

File: content/50.discussion.md [hunk 6]
```diff
@@ -1,81 +1,102 @@
...
+A range of other antivirals are also in development.
 Development of antivirals will be further facilitated as research reveals more information about the interaction of SARS-CoV-2 with the host cell and host cell genome, mechanisms of viral replication, mechanisms of viral assembly, and mechanisms of viral release to other cells; this can allow researchers to target specific stages and structures of the viral life cycle.
-
-Antibodies against viruses, also known as antiviral monoclonal antibodies, could be an alternative as well and are described in detail in an above section.
+Many researchers have also focused their attention to the potential use of dietary supplements and nutraceuticals.
+Indeed, there has been recent interest for the potential use of vitamin D as a prophylactic and therauetic agent against COVID-19 as several observational studies have linked low vitamin D status to its incidence [@doi:10.1111/joim.13149; @doi10.1001/jamanetworkopen.2020.19722].
```
Comment (RLordan) [inline content/50.discussion.md:68]:
```suggestion
Indeed, there has been recent interest for the potential use of vitamin D as a prophylactic and therauetic agent against COVID-19 as several observational studies have linked low vitamin D status to its incidence [@doi:10.1111/joim.13149; @doi:10.1001/jamanetworkopen.2020.19722]. 
```

**Pull request merged**